backend/src/announcement/announcement.service.ts

```diff
@@ -36,42 +41,77 @@ export class AnnouncementServices {
     }
   }
   public async createAnnouncement(
+    user: User,
     data: AnnouncementDTO
   ): Promise<AnnouncementDTO> {
     try {
       const announcement = await this.prisma.announcement.create({
         data,
       });
+      this.eventEmitter.emit('create.logs', {
+        entityName: 'announcement',
+        action: ActionENUM.ADDED,
+        username: user.name,
+        priority: PriorityENUM.IMPORTANT,
```

Comment:
as per @noice-noise, all entities aside from the `user` entity will have an enum priority of `IMPORTANT` for now. :D (for testing purposes sad sa socket.io nga toasts XD)